src/core/binformat.js

```diff
@@ -423,7 +423,7 @@ exports.ter = {
   tecNO_LINE_REDUNDANT     : 127,
   tecPATH_DRY              : 128,
   tecUNFUNDED              : 129,  // Deprecated, old ambiguous unfunded.
-  tecMASTER_DISABLED       : 130,
+  tecNO_ALTERNATIVE_KEY    : 130,
```

Comment:
it wont fail right? just the output will be tecNO_ALTERNATIVE_KEY when it used to say tecMASTER_DISABLED?